docs/guides/runtime-options-overview.mdx

```diff
@@ -18,288 +19,298 @@ service = QiskitRuntimeService(channel="ibm_cloud", channel_strategy="q-ctrl")
 
-In the example below, we create an instance of the `Options` class. `optimization_level` is a first-level option and can be passed as an input parameter. Options related to the execution environment are passed using the `environment` parameter.
+<span id="options-defaults"></span>
+### Defaults
 
-```python
-from qiskit_ibm_runtime import Options
+For V2 primitives, if you do not specify a value for an option, it is given a special value of `Unset` and the server default value is used. Thus, the default value will be the same regardless of your code version.
+For V1 primitives, if you do not specify a value for an option, the Qiskit Runtime client's default value is used. In V1, the default value is dependant on the code version.
 
-options = Options(optimization_level=1, environment={"log_level": "INFO"})
-```
-
-The `Options` class supports auto-complete. Once you create an instance of the `Options` class, you can use auto-complete to see what options are available. If you choose one of the categories, you can use auto-complete again to see what options are available under that category.
+The tables in the [Options classes summary](#options-classes) section lists the default values.
 
-```python
-from qiskit_ibm_runtime import Options
+<span id="options-precedence"></span>
+### Precedence rules
 
-options = Options()
-options.resilience_level = 1
-options.execution.shots = 2048
-```
-## Pass options to a primitive
+There are several ways to set options.  If you set different values for the same option in multiple places, this is the order of precedence:
 
-### Options class
+1. Any options set after primitive initialization.
+2. Any options set during primitive initialization.
 
-When creating an instance of the `Estimator` or `Sampler` class, you can pass in the `options` you created in the options class. Those options will then be applied when you use `run()` to perform the calculation.  Example:
+<Admonition type="caution" title="Important">
+Shots (Sampler) or precision (Estimator) set in `run()` are not considered options, but take precidence over values set for `default_shots`or `default_precision`. See the [Run() method](specify-runtime-options#run-method) section for full details.
+</Admonition>
 
-<Tabs>
-  <TabItem value="PrimV2" label="V2 primitives">
-    `SamplerV2` and `EstimatorV2` have separate options classes that do not need to be instantiated.  You can see the available options and update option values during or after primitive initialization.  Those options will then be applied when you use `run()` to perform the calculation.  Example:
+In the following example, these values would be used for the `estimator.run` job: `shots = 128` and `resilience_level = 1`.
```

Comment:
The code doesn't actually change the resilience level; it just turns on meas mitigation. 

```suggestion
In the following example, these values would be used for the `estimator.run` job: `shots = 128` and `measure_mitigation = True`.
```